src/wp-includes/script-loader.php

```diff
@@ -1302,7 +1302,7 @@ function wp_default_scripts( $scripts ) {
 		$scripts->add( 'wp-color-picker', "/wp-admin/js/color-picker$suffix.js", array( 'iris' ), false, 1 );
 		$scripts->set_translations( 'wp-color-picker' );
 
-		$scripts->add( 'dashboard', "/wp-admin/js/dashboard$suffix.js", array( 'jquery', 'admin-comments', 'postbox', 'wp-util', 'wp-a11y' ), false, 1 );
+		$scripts->add( 'dashboard', "/wp-admin/js/dashboard$suffix.js", array( 'jquery', 'admin-comments', 'postbox', 'wp-util', 'wp-a11y', 'wp-date', 'wp-i18n' ), false, 1 );
```

Comment:
```suggestion
		$scripts->add( 'dashboard', "/wp-admin/js/dashboard$suffix.js", array( 'jquery', 'admin-comments', 'postbox', 'wp-util', 'wp-a11y', 'wp-date' ), false, 1 );
		$scripts->set_translations( 'dashboard' );
```

Reply:
Is the reason for this that it's already a dependency of `wp-ally`, so it's not technically necessary to list it explicitly?

Reply:
No, `WP_Scripts::set_translations()` takes care of adding the required dependencies, currently `wp.i18n`.

Reply:
Ah, cool 👍🏻